src/components/films.jsx

```diff
@@ -0,0 +1,90 @@
+
+import React, { useState, useEffect, useCallback } from "react";
+
+import Movie from "./Movie";
+
+import '../App.css';
+
+function Films() {
+  const [value, setValue] = useState(1);
+  const [items, setItems] = useState([]);
+  const [isLoading, setLoading] = useState(false);
+  const [isEmpty, setEmpty] = useState(false);
+  const [isError, setError] = useState(false);
+
+  useEffect(() => {
+    fetchFilms()
+  }, [value]);
+
+  function changeSelect(event) {
+    if (event.target.value === "Минск") {
+      setValue(1);
+
+      return setValue(1);
+    } else {
+
+      return (setValue(5))
+    }
```

Comment:
```suggestion
    if (event.target.value === "Минск") {
      setValue(1);

      return setValue(1);
    } else {

      return (setValue(5))
    }
```

зачем здесь return?
переписать в 1 строчку